src/snapshots/records/record_access_function_call.md

```diff
@@ -0,0 +1,35 @@
+# META
+~~~ini
+description=Record field access with function calls (dot-access)
+type=expr
+~~~
+# SOURCE
+~~~roc
+person.transform(42)
```

Comment:
```suggestion
(person.transform)(42)
```

Currently this is using static dispatch syntax; this suggestion changes the syntax to reflect what the description says it's doing 😄